core/ibft.go

```diff
@@ -124,13 +126,17 @@ func (i *IBFT) startRoundTimer(ctx context.Context, round uint64) {
 	defer i.wg.Done()
 
 	var (
-		duration     = int(i.baseRoundTimeout)
-		roundFactor  = int(math.Pow(float64(2), float64(round)))
-		roundTimeout = time.Duration(duration * roundFactor)
+		duration        = int(i.baseRoundTimeout)
+		roundFactor     = int(math.Pow(float64(2), float64(round)))
+		roundTimeout    = time.Duration(duration * roundFactor)
+		epochAdditional = 0 * time.Second
 	)
 
 	//	Create a new timer instance
-	timer := time.NewTimer(roundTimeout + i.additionalTimeout)
+	if i.backend.IsEpochHeight() {
+		epochAdditional = i.additionalEpochTimeOut
+	}
+	timer := time.NewTimer(roundTimeout + i.additionalTimeout + epochAdditional)
```

Comment:
could think of a way to have a separated variable to sum up all these for readability